tools/c7n_azure/tests_azure/tests_resources/test_advisor.py

```diff
@@ -0,0 +1,26 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from ..azure_common import BaseTest
+
+
+class AdvisorTest(BaseTest):
+    def test_azure_advisor_schema_validate(self):
+        p = self.load_policy({
+            'name': 'test-azure-advisor',
+            'resource': 'azure.advisor'
+        }, validate=True)
+        self.assertTrue(p)
+
+    def test_find_by_name(self):
+        p = self.load_policy({
+            'name': 'test-azure-advisor',
+            'resource': 'azure.advisor',
+            'filters': [
+                {'type': 'value',
+                 'key': 'name',
+                 'op': 'glob',
+                 'value_type': 'normalize',
+                 'value': 'ccadvisor*'}],
```

Comment:
Can you please add an ARM template to deploy the resource to the PR?

How long is the provisioning time/any special permissions required to deploy it? (The only reason I'm asking is to ensure it can be a part of live tests and not only recorded test)

Reply:
Unlike other resources, Azure Advisor recommendations cannot be created via CLI/REST/Portal. Advisor recommendations are built-in feature. They can be either enabled or disabled. It doesn't have ARM deployment template. 

For your reference:
https://docs.microsoft.com/en-us/rest/api/advisor/recommendations
https://docs.microsoft.com/en-us/azure/templates/microsoft.advisor/recommendations/suppressions?tabs=json
https://docs.microsoft.com/en-us/cli/azure/advisor/recommendation?view=azure-cli-latest#az_advisor_recommendation_enable